### PR TITLE
Breaking sentences into new lines, removing wording, bolding questions

### PR DESCRIPTION
**Pull request by krainboltgreene**
- Breaking all paragraphs into new-line-per-sentence for readability while editing
- Cleaning up the wording, removing commas
- Formatting lists to be easier to read

### CODE REVIEW

**Comment by joeellis** [comment]
Weird, I merged this but it didn't close, did I miss something?

**Comment by krainboltgreene** [comment]
Me and @bhuga both edited the README.

**Comment by joeellis** [comment]
Yea, but I got his changes in, then I could have sworn I merged yours in.  Anyways, I'll figure it out tonight and make sure this gets closed properly

**Comment by krainboltgreene** [comment]
This is on me. I'll have an update in 20m.

**Comment by joeellis** [comment]
K, I think Brad had some changes as well last night, so watch out for those

File: README.markdown [hunk 1]
```diff
@@ -1,38 +1,49 @@
 WELCOME TO BARCAMPNOLA 2012 WEBSITE
 =============
 
-Hello, and thank you for possibly thinking about contributing to our BarCampNOLA 2012 website.  The whole goal of this project is to try and get people to contribute to a larger project so they can feel happy and proud of themselves that they made something great.  The other goal is to use a lot of dinosaurs and video game text for seemingly no reason.
+Hell and thank you for thinking about contributing to our BarCampNOLA 2012 website.
```
Comment (coryf) [inline README.markdown:4]: This should probably be "Hello" instead of "Hell".

**Comment by jwietelmann** [comment]
@krainboltgreene Might be easier at this point to just close this PR and do another one. The README has already diverged pretty far from the version you edited.

**Pull request closed without merging**